editor/src/clj/editor/gui.clj

```diff
@@ -661,11 +746,190 @@
         :type type ; Explicitly include the type (pb-field is optional, so :type-box would be stripped otherwise).
         :child-index child-index))) ; Used to order sibling nodes in the SceneDesc.
 
+;; SDK api
+(defmacro layout-property-getter [prop-sym]
+  {:pre [(symbol? prop-sym)]}
+  (let [prop-kw (keyword prop-sym)]
+    `(g/fnk [~'prop->value ~prop-sym]
+       (get ~'prop->value ~prop-kw ~prop-sym))))
```

Comment:
Layout-specific overrides are now stored in a `:layout->prop->override` map in the `GuiNodes`. We produce the `prop->value` output from it and the `current-layout`, merging in the overrides from override originals recursively to produce the final map. However, if we're using the default layout, we can immediately return `nil` from the `prop->value` output, and the raw property value of the `prop-sym` will be used.